dev/com.ibm.ws.springboot.support_fat/fat/src/com/ibm/ws/springboot/support/fat/SSLTests20.java

```diff
@@ -26,11 +27,26 @@
 @Mode(FULL)
 public class SSLTests20 extends SSLCommonTests {
 
+    @After
+    public void stopTestServer() throws Exception {
+        String methodName = testName.getMethodName();
+        if (methodName != null && methodName.contains(DEFAULT_HOST_WITH_APP_PORT)) {
+            super.stopServer(true, "CWWKT0015W");
+        } else {
+            super.stopServer();
```

Comment:
why not `stopServer(true);`

Reply:
super.stopServer() will call super.stopServer(true)